packages/mdx-embed/src/components/tiktok/tiktok.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen, act } from '@testing-library/react';
+
+import { TikTok } from './';
+
+describe('<TikTok />', () => {
+  beforeEach(() => {
```

Comment:
@PaulieScanlon I've not added this before each on the tests I've done, should I retro them in?

@kriswep this looks fine to me 👍